seednode/README.md

```diff
@@ -17,3 +19,47 @@ Follow the logs created by the service by inspecting
 ```
 journalctl --unit bisq-seednode --follow
 ```
+
+
+## Running using docker (experimental)
+
+The production docker image uses the build dir, so make sure you have built the appropriate tag/branch.
+Here's the checklist to start your docker seednode:
+
+- build the seednode code: `./gradlew build`
```

Comment:
This step should be part of image build process and build directory should be excluded.

Reply:
that would indeed be better, it used to be like that but the repos and branches kept changing :)

I could pass in the repo+branch via the .env file and checkout/build using that info. Of course that implies checking it out every time. Other suggestions welcome.